py-rse/ci.Rmd

```diff
@@ -287,47 +251,49 @@ after_success:
 - codecov
 ```
 
+We use the Travis setting `after_success` since the coverage tool is a separate
+service and the Travis build should happen first.
```

Comment:
what does "separate service" mean? (will readers know?)

Reply:
Fixed language to be simpler and shorter.